src/main/java/com/disi/TravelPoints/controller/WishlistController.java

```diff
@@ -6,6 +6,7 @@
 import lombok.RequiredArgsConstructor;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
```

Comment:
I was wondering if you need the last 3 imports since you import all here with *

Reply:
xvdsvdsvds

Reply:
Sorry, resulted after resolving conflicts

Reply:
It's okay, you don't need to say sorry